DependencyInjection/EzPlatformUIExtension.php

```diff
@@ -45,6 +45,11 @@ public function prepend(ContainerBuilder $container)
         $this->prependYui($container);
         $this->prependCss($container);
         $this->prependImageVariations($container);
+
+        $configFile = __DIR__ . '/../Resources/config/kernel.yml';
```

Comment:
I don't think I'd name this file `kernel.yml`. The standard for defining views is `views.yml`, could you rename it ?